app/package.json

```diff
@@ -2,13 +2,21 @@
   "name": "@klimadao/app",
   "private": true,
   "scripts": {
-    "dev": "next dev -p 3001",
+    "dev": "npm run lingui:compile && next dev -p 3001",
     "build": "next build",
     "start": "next start",
     "export": "next export",
-    "lint": "next lint"
+    "lint": "next lint",
+    "lingui:extract": "NODE_ENV=test lingui extract",
+    "lingui:compile": "NODE_ENV=test lingui compile",
+    "postinstall" : "npm run lingui:compile"
   },
   "dependencies": {
+    "@babel/core": "^7.16.0",
```

Comment:
are you sure we need babel, doesn't lingui and nextjs have it as a subdependency already?

Reply:
Good catch. I was following installation instructions blindly.